scripts/clear_testing.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Clear out our local testing directories."""
+import argparse
+import os
+import shutil
+
+TESTING_DIRS = [
+    "local_providers/aws_local",
+    "local_providers/aws_local_0",
+    "local_providers/aws_local_1",
+    "local_providers/aws_local_2",
+    "local_providers/aws_local_3",
+    "local_providers/aws_local_4",
+    "local_providers/aws_local_5",
+    "local_providers/azure_local",
+    "local_providers/insights_local",
+    "pvc_dir/insights_local",
+    "pvc_dir/processing",
+]
+
+
+def main(*args, **kwargs):
+    testing_path = kwargs["testing_path"]
+
+    paths_to_clear = [f"{testing_path}/{directory}" for directory in TESTING_DIRS]
+
+    for path in paths_to_clear:
+        try:
+            print(f"Checking {path}")
+            dirs_to_remove = [f.path for f in os.scandir(path) if f.is_dir()]
+            for directory in dirs_to_remove:
+                print(f"Removing {directory}")
+                shutil.rmtree(directory)
```

Comment:
If you're using `rmtree`, you only need to remove the `local_providers` and `pvc_dir` directories. It'll recursively remove everything in those directories.

Second, you may want to add a call to `os.mkdir()` to recreate the removed paths.

Reply:
Each of those directories gets mounted as a volume, so I'd like to not delete them, just their contents.

Reply:
The most direct option is `git clean -- <path>`

These paths are in the repository, so you can have git do all of the work for you without needing to specify anything beyond the base `testing` path.